code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -685,3 +685,14 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 	merge_type = /obj/item/stack/sheet/meat
 	material_type = /datum/material/meat
 	material_modifier = 1 //None of that wussy stuff
+
+/obj/item/stack/sheet/obamium
+	name = "obamium sheet"
+	desc = "Finally...Obamium"
+	singular_name = "obamium sheet"
+	icon_state = "sheet-meat"
```

Comment:
add a sprite for this

Reply:
it's a greyscale sheet its fine

Reply:
yeah but you could make it the rotating pyramid from the meme